development/image-syncer/external-images.yaml

```
@@ -14,6 +14,7 @@ images:
 # Golang image is pinned because it is force updated in the upstream
 - source: "golang@sha256:f28579af8a31c28fc180fb2e26c415bf6211a21fb9f3ed5e81bcdbf062c52893"
   tag: "1.17.6-alpine3.15"
+- source: "golang/1.17.7-alpine3.15"
```

We pin Golang version since it changes on the upstream Docker HUB, and we want to have unmutable tags on our side.
In case of update you can update the SHA for linux/arm64 image and retag the image on our mirror (e.g. "1.17.7-alpine3.15-v1")
Replacing `1.17.6-alpine3.15` won't remove it from our mirror
```suggestion
- source: "golang@sha256:4e84e7209cf9d803fdb936260e37811a09c2b5107a1cc88db1bf6f1c9f3be9eb"
  tag: "1.17.7-alpine3.15"
```